extensions/amp-access/amp-access-analytics.md

```diff
@@ -143,3 +143,19 @@ The `access-login-failed` event is issued when Login dialog has failed due to an
   }
 }
 ```
+
+#### Access analytics variables
+
+Access contributes the following URL substitutions to the [amp-var-substitutions.md](spec/amp-var-substitutions.md).
+
+###### ACCESS_READER_ID
+
+The `ACCESS_READER_ID` variable is substituted with the Reader ID used for access operations. This is a relatively sensitive ID and care should be taken to avoid sharing it with 3rd party analytics packages.
```

Comment:
+1. Suggested reword: "This is a sensitive ID and care should be taken in sharing it. Please ensure that any data you pass to a third party complies with its terms of service."